src/planner/binder/statement/bind_simple.cpp

```diff
@@ -17,6 +17,16 @@
 
 namespace duckdb {
 
+unique_ptr<LogicalOperator> DuckCatalog::BindAlterAddIndex(Binder &binder, TableCatalogEntry &table_entry,
+                                                           unique_ptr<LogicalOperator> plan,
+                                                           unique_ptr<CreateIndexInfo> create_info,
+                                                           unique_ptr<AlterTableInfo> alter_info) {
+	D_ASSERT(plan->type == LogicalOperatorType::LOGICAL_GET);
+	IndexBinder index_binder(binder, binder.context);
+	return index_binder.BindCreateIndex(binder.context, std::move(create_info), table_entry, std::move(plan),
+	                                    std::move(alter_info));
+}
```

Comment:
Sadly, I could not unify this with `BindCreateIndex`, as that one expects a `CreateStatement`, whereas now we get an `AlterStatement`.

```cpp
	virtual unique_ptr<LogicalOperator> BindCreateIndex(Binder &binder, CreateStatement &stmt, TableCatalogEntry &table,
	                                                    unique_ptr<LogicalOperator> plan) = 0;
```